main/integration-tests/__test__/api-tests/workflow-drafts/publish-workflow-draft.test.js

```diff
@@ -0,0 +1,111 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('Publish workflow drafts scenarios', () => {
+  let setup;
+  let adminSession;
+
+  // Creates a publish request object that is ready to be used for the publish operation
+  const createPublishRequest = async () => {
+    const workflowId = setup.gen.string({ prefix: 'wf-publish-test-update' });
```

Comment:
nit: How about keeping the name pattern consistent with the rest of the tests (`<verb>-<resource>-test`) ?